lbz/authz.py

```diff
@@ -22,24 +22,27 @@
 
 
 class Authorizer:
-    def __init__(self, auth_jwt: str, resource_name: str, permission_name: str):
+    def __init__(self, auth_jwt: str, resource_name: str, permission_name: str, auth_scope: dict = None):
```

Comment:
I'd rename `auth_scope` -> `policy`, or maybe `policy_override` to be clear that this overrides the policy from the jwt